include/utils/mqtt_abstraction_impl.hpp

```diff
@@ -80,6 +80,10 @@ class MQTTAbstractionImpl {
     /// \brief unsubscribes from the given \p topic
     void unsubscribe(const std::string& topic);
 
+    ///
+    /// \brief clears any previously published topics that had the retain flag set
+    void clear_retained_topics();
```

Comment:
Is there any documentation on how or when the `retain-topics` flag should be used?